bundle/config/mutator/load_git_details.go

```diff
@@ -40,20 +40,12 @@ func (m *loadGitDetails) Apply(ctx context.Context, b *bundle.Bundle) diag.Diagn
 		log.Warnf(ctx, "failed to load current branch: %s", err)
 	}
 
-	// load commit hash if undefined
-	if b.Config.Bundle.Git.Commit == "" {
-		commit, err := repo.LatestCommit()
-		if err != nil {
-			log.Warnf(ctx, "failed to load latest commit: %s", err)
-		} else {
-			b.Config.Bundle.Git.Commit = commit
-		}
-	}
-	// load origin url if undefined
-	if b.Config.Bundle.Git.OriginURL == "" {
-		remoteUrl := repo.OriginUrl()
-		b.Config.Bundle.Git.OriginURL = remoteUrl
+	commit, err := repo.LatestCommit()
+	if err != nil {
+		log.Warnf(ctx, "failed to load latest commit: %s", err)
 	}
+	b.Config.Bundle.Git.Commit = commit
+	b.Config.Bundle.Git.OriginURL = repo.OriginUrl()
```

Comment:
LGTM, but can you also add the `bundle: "readonly"` tag to b.Config.Bundle.Git.OriginURL? That'll remove it from the bundle schema as well.

Reply:
thanks, done!